tools/background_removal/background_removal.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="background_removal" name="Remove image background" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.05">
+        <description>with scikit-image</description>  
+    <macros>  
+        <import>creators.xml</import>
+        <import>tests.xml</import>
+        <token name="@TOOL_VERSION@">0.24.0</token>  
+        <token name="@VERSION_SUFFIX@">0</token>  
+    </macros>  
+    <creator>
+        <expand macro="creators/rmassei"/>
+    </creator>
+    <requirements>
+        <requirement type="package" version="0.24.0">scikit-image</requirement>
+    </requirements>
+    <command detect_errors="aggressive">
+        <![CDATA[
+        python "$__tool_directory__/background_removal.py" $input_image $filter $radius $output
+        ]]>
+    </command>
+    <inputs>
+        <param name="input_image" type="data" format="tiff, jpg, jpeg, png, tif" label="Input Image"/>
+        <param name="filter" type="select" label="Select Filter">
+            <option value="rolling_ball">Rolling-Ball Algorithm</option>
+            <option value="dog">Difference of Gaussians</option>
+            <option value="top_hat">Top-Hat Filter</option>
+        </param>
+        <param name="radius" type="float" label="Radius" value="20"/>
+    </inputs>
+    <outputs>
+        <data name="output" format="tiff" label="Background substraction output"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="input_image" value="img.jpg"/>
+            <param name="filter" value="rolling_ball"/>
+            <param name="radius" value="20"/>
+            <expand macro="tests/intensity_image_diff" name="output" value="img_output_rb.tif" ftype="tiff"/>
```

Comment:
There is no such file `img_output_rb.tif`?

You probably meant `out_rolling_ball.tif`?
```suggestion
            <expand macro="tests/intensity_image_diff" name="output" value="out_rolling_ball.tif" ftype="tiff"/>
```

However, I noticed that `img.jpg` is an 8bit image, but `out_rolling_ball.tif` is 16bits. As a consequence, the range of values is not preserved. This is unconvenient in practice, [right](https://github.com/BMCV/galaxy-image-analysis/issues/105#issuecomment-1994180241)?